doc/STYLE_GUIDE.md

```diff
@@ -19,39 +18,37 @@
   * Use gender-neutral pronouns and gender-neutral plural nouns.
     * OK: _they_, _their_, _them_, _folks_, _people_, _developers_
     * NOT OK: _his_, _hers_, _him_, _her_, _guys_, _dudes_
-* When combining wrapping elements (parentheses and quotes), terminal
-  punctuation should be placed:
+* When combining wrapping elements (parentheses and quotes), place terminal
+  punctuation:
   * Inside the wrapping element if the wrapping element contains a complete
-    clause — a subject, verb, and an object.
+    clause.
   * Outside of the wrapping element if the wrapping element contains only a
     fragment of a clause.
 * Documents must start with a level-one heading.
 * Prefer affixing links to inlining links — prefer `[a link][]` to
   `[a link](http://example.com)`.
-* When documenting APIs, note the version the API was introduced in at
-  the end of the section. If an API has been deprecated, also note the first
-  version that the API appeared deprecated in.
-* When using dashes, use [Em dashes][] ("—" or `Option+Shift+"-"` on macOS)
-  surrounded by spaces, as per [The New York Times Manual of Style and Usage][].
+* When documenting APIs, update the YAML comment associated with the API as
+  appropriate. This is especially true when introducing or deprecating an API.
```

Comment:
AFAIK we have no documentation on the YAML comments, so it's guesswork based on the existing YAML comments as to what should go in there when introducing a new API. Kind of falls under the general: https://github.com/nodejs/node/issues/25548

Reply:
I suppose at least this is an improvement because it actually *mentions* the YAML comments.